lib/api/reset-password.js

```diff
@@ -102,6 +103,32 @@ module.exports = function(app, auth) {
     });
   }
 
+  // Reset admin password
+  function resetAdminPassword(req, res, next) {
+    auth.session(req, res, function() {
+      User.findOne({ username: req.user.username }, function(err, user) {
+        if (err) return next(err);
+        if (!user || user.username != 'admin') return next(new Error('Not an admin'));
+        user.password = req.body.password;
+        user.hasDefaultPassword = false;
+        user.save(function(err, user) {
+          res.cookie(auth.stubKey, 'yes', auth.stubCookie);
+          if (err) {
+            console.log(err);
```

Comment:
Nothing atypical. I will change to same "template" as in other save calls in the API